rest-api-spec/src/main/resources/rest-api-spec/api/inference.inference.json

```diff
@@ -0,0 +1,45 @@
+{
+  "inference.inference": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/post-inference-api.html",
+      "description": "Perform inference"
+    },
+    "stability": "stable",
+    "visibility": "public",
+    "headers": {
+      "accept": ["application/json"],
+      "content_type": ["application/json"]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_inference/{inference_id}",
```

Comment:
I think in most places we're directing users to include the `task_type` in the URL. Would it cause problems if we remove this path?